mne/io/meas_info.py

```diff
@@ -1861,16 +1867,19 @@ def _force_update_info(info_base, info_target):
             i_targ[key] = val
 
 
-def anonymize_info(info):
+def anonymize_info(info, daysback=None, keep_his=False):
     """Anonymize measurement information in place.
 
-    Reset 'subject_info', 'meas_date', 'file_id', and 'meas_id' keys if they
-    exist in ``info``.
-
```

Comment:
Rather than just deleting this it should probably be moved to a new Notes section. It's important to say what you do, including the new stuff -- for example, nuking `raw.info['description']` could mess up people who use this field for non-subject-specific stuff. If you tell them that you do it, it can avoid some unpleasant surprises.